chromium_src/chrome/browser/ui/webui/webui_util.cc

```diff
@@ -37,16 +27,4 @@ void SetupWebUIDataSource(content::WebUIDataSource* source,
       network::mojom::CSPDirectiveName::ScriptSrc, kBraveCSP);
 }
 
-#if BUILDFLAG(OPTIMIZE_WEBUI)
-void SetupBundledWebUIDataSource(content::WebUIDataSource* source,
-                                 base::StringPiece bundled_path,
-                                 int bundle,
-                                 int default_resource) {
-  SetupBundledWebUIDataSource_ChromiumImpl(source, bundled_path, bundle,
-                                           default_resource);
-  source->OverrideContentSecurityPolicy(
```

Comment:
where does this happen now?

Reply:
I think `webui::SetupWebUIDataSource` is always used now for optimized and not, and we have the same call in the override there.